packages/core/src/__tests__/browser.test.tsx

```diff
@@ -2,16 +2,20 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { styled } from '@compiled/core';
 
+jest.mock('../../../runtime/dist/is-node', () => ({
+  isNodeEnvironment: () => false,
```

Comment:
not my proudest code. because this isn't official api in the runtime we need to target dist directly.